scripts/blocklist-skylink.sh

```diff
@@ -3,14 +3,49 @@
 set -e # exit on first error
 
 if [ -z "$1" ]; then
-    echo "Please provide a skylink to blocklist" && exit 1
+    echo "Please provide either a skylink or file with skylinks separated by blank lines" && exit 1
```

Comment:
(nit) "separated by blank lines" is confusing

Reply:
it should say new lines, my bad :)